src/common/base-store.ts

```diff
@@ -36,8 +36,12 @@ export class BaseStore<T = any> extends Singleton {
     return path.basename(this.storeConfig.path);
   }
 
-  get syncChannel() {
-    return `store-sync:${this.name}`
+  get syncRendererChannel() {
```

Comment:
Why we need this separation in event-sync btw main/renderer?

Reply:
`broadcastMessage` would send same message in a loop if we use same channel for both directions (because broadcast message will get delivered also to the same process).